### ISSUE TITLE
some missing attributes in config.py

### ISSUE THREAD

**Issue by yl1991** [louis-she/sfd.pytorch#20]
hi Louis,
I think there are some attributes in config.py such as RANDOM_CROP, RANDOM_FLIP ,MIN_CROPPED_RATIO, etc. 
Could you please share the config.py file you used for training? 
Many thanks,
Yiming

**Comment by louis-she** [comment]
I’ll merge the newest code to master later in today. Recently I’m doing experiments on some tricks trying to improve the mAP, but unfortunately the model do not converge anymore which drives me crazy. I’m still doing experiments on the branch `lab` and I will merge it into master. If you are going to train the model, it should be fine tuning to make it converge.

**Comment by louis-she** [comment]
Hi @yl1991 ,
I have updated the `config.py.example` and it's  now ready to be trained. But the model in the `README.md` is useless because I have changed the model(added some new layer). A new model is training now and when it converged, I'll upload the new model weights file.

**Comment by yl1991** [comment]
Hi @louis-she 
Thanks for answering and working on this. Unfortunately the losses exploded and became Nan after 5 iterations. Do you think its because of the new layers?

> INFO:root:[train][epoch:1][iter:0][total:3220] loss_class 12.18820953 - loss_reg 2.41406178 - total 14.60227108
> INFO:root:[train][epoch:1][iter:1][total:3220] loss_class 119.29055786 - loss_reg 6.50044680 - total 125.79100800
> INFO:root:[train][epoch:1][iter:2][total:3220] loss_class 211.55720520 - loss_reg 89.25713348 - total 300.81433105
> INFO:root:[train][epoch:1][iter:3][total:3220] loss_class 30.36459923 - loss_reg 121.55815887 - total 151.92276001
> INFO:root:[train][epoch:1][iter:4][total:3220] loss_class 7628.62548828 - loss_reg 90.47879028 - total 7719.10449219
> INFO:root:[train][epoch:1][iter:5][total:3220] loss_class 283386539147264.00000000 - loss_reg 6571166793728.00000000 - total 289957704892416.00000000
> INFO:root:[train][epoch:1][iter:6][total:3220] loss_class nan - loss_reg nan - total nan
> INFO:root:[train][epoch:1][iter:7][total:3220] loss_class nan - loss_reg nan - total nan
> INFO:root:[train][epoch:1][iter:8][total:3220] loss_class nan - loss_reg nan - total nan
> INFO:root:[train][epoch:1][iter:9][total:3220] loss_class nan - loss_reg nan - total nan
> INFO:root:[train][epoch:1][iter:10][total:3220] loss_class nan - loss_reg nan - total nan
> 


Best,
Yiming

**Comment by louis-she** [comment]
@yl1991 have you try to pull down the newest master code? everything works fine in my machine
![image](https://user-images.githubusercontent.com/2050207/43143848-61f91b04-8f8e-11e8-9147-380b290531a3.png)

**Comment by yl1991** [comment]
Oh my bad it is working now :) cheers.

**Comment by louis-she** [comment]
hi @yl1991 ,
now that you are training the model too, I suggest that you change some configuration. Now I'm training it with default configuration. I suggest you use the following:
```python
...
RANDOM_CROP = True
...
RANDOM_FLIP = False
```

**Comment by yl1991** [comment]
Sure I can train with different parameters to see the performance

**Comment by yl1991** [comment]
random crop takes too much GPU memory as I only got one 1080ti. I am training with random flip

**Comment by louis-she** [comment]
And , I have ignore the first prediction feature map, which is the output of `Conv3_3` layer. It's been designed to detect very small faces. If you wanna add it back, change the configuration to:
```python
    ANCHOR_STRIDE = [4, 8, 16, 32, 64, 128]
    ANCHOR_SIZE = [16, 32, 64, 128, 256, 512]
```
and then uncomment the the 2 line of code in https://github.com/louis-she/sfd.pytorch/blob/master/model.py#L86

**Comment by yl1991** [comment]
Lol Very interested in this paper, it would be great if I could contribute.

**Comment by louis-she** [comment]
Cool.  Here you can monitor the training process on my machine:
https://187b3cfc.ngrok.io/#scalars&_smoothingWeight=0.99

**Comment by louis-she** [comment]
The mAP is 0.3 around and the loss is still decreasing, I'm going to go to sleep now.
